### PR TITLE
Install pywatchman as a local/development requirement.

### PR DESCRIPTION
**Pull request by iloveagent57**

### CODE REVIEW

File: requirements/production.txt [hunk 1]
```diff
@@ -64,7 +64,7 @@ edx-opaque-keys==2.0.2    # via -r requirements/base.in, edx-ccx-keys, edx-drf-e
 edx-rest-api-client==5.1.0  # via -r requirements/base.in
 elasticsearch==1.9.0      # via -c requirements/constraints.txt, -r requirements/base.in
 future==0.18.2            # via django-ses, pyjwkest
-gevent==1.5.0             # via -r requirements/production.in
+gevent==20.4.0            # via -r requirements/production.in
```
Comment (Dillon-Dumesnil) [inline requirements/production.txt:67]: Well this looks scary. Oh they probably do date releases? Would you mind just doing a quick double check of this and make sure that we aren't actually hopping 19 major versions.

**Review by Dillon-Dumesnil — APPROVED** [review]
One kinda scary bit. Otherwise, looks great.

**Comment by Dillon-Dumesnil** [comment]
I restarted the failing build for ya. Looked like a flaky test

File: requirements/production.txt (hunk 1 above)
Comment (mikix) [reply requirements/production.txt:67]: https://pypi.org/project/gevent/#history

Looks like they just switched versioning schemes.

**Review by mikix — APPROVED** [review]

File: requirements/production.txt (hunk 1 above)
Comment (iloveagent57) [reply requirements/production.txt:67]: 👍 thanks for checking!  And thanks for re-starting tests, Dillon.

**Review by feanil — APPROVED** [review]

**Pull request merged**